pkg/scheduler/framework/plugins/tainttoleration/taint_toleration.go

```diff
@@ -56,10 +58,35 @@ func (pl *TaintToleration) Name() string {
 // failed by this plugin schedulable.
 func (pl *TaintToleration) EventsToRegister() []framework.ClusterEventWithHint {
 	return []framework.ClusterEventWithHint{
-		{Event: framework.ClusterEvent{Resource: framework.Node, ActionType: framework.Add | framework.Update}},
+		{Event: framework.ClusterEvent{Resource: framework.Node, ActionType: framework.Add | framework.UpdateNodeTaint}, QueueingHintFn: pl.isSchedulableAfterNodeChange},
```

Comment:
should be OK to change it to `UpdateNodeTaint` based on the discussion in -
https://github.com/kubernetes/kubernetes/pull/122292

Reply:
PreCheck includes not only taint validation, also like node affinity etc., so is this enough? For example, a new added node was rejected by nodeAffinity, but then updated the node label.

Reply:
Under this case, maybe it's taint tolerated before.

Reply:
You're actually right; if we want to cover all Nodes that possibly filtered out by preCheck previously, we have to keep Node/Update and actually add Pod/Delete (for node port).

But, from the discussion in https://github.com/kubernetes/kubernetes/pull/122292/files#r1495008072 and then we only added NodeTaint after all, I understand we agree to ignore those cases because they're very minor scenarios, and more importantly, that'd be a big degradation.

Reply:
I don't think we can apply the same argument. In the other PR, Node label updates have always been ignored.

In this case, we could be introducing a regression.

Let's keep Update until we finish migrating all the preChecks.